examples/large_models/deepspeed/opt/custom_handler.py

```diff
@@ -0,0 +1,114 @@
+import logging
+from abc import ABC
+
+import torch
+import transformers
+from transformers import AutoModelForCausalLM, AutoTokenizer
+
+from ts.context import Context
+from ts.torch_handler.distributed.base_deepspeed import BaseDeepSpeedHandler
+
+logger = logging.getLogger(__name__)
+logger.info("Transformers version %s", transformers.__version__)
+
+
+class TransformersSeqClassifierHandler(BaseDeepSpeedHandler, ABC):
+    """
+    Transformers handler class for sequence, token classification and question answering.
+    """
+
+    def __init__(self):
+        super(TransformersSeqClassifierHandler, self).__init__()
+        self.max_length = None
+        self.tokenizer = None
+        self.initialized = False
+
+    def initialize(self, ctx: Context):
+        """In this initialize function, the HF large model is loaded and
+        partitioned using DeepSpeed.
+        Args:
+            ctx (context): It is a JSON Object containing information
+            pertaining to the model artefacts parameters.
+        """
+        model_dir = ctx.system_properties.get("model_dir")
+        self.max_length = int(ctx.model_yaml_config["handler"]["max_length"])
+        self.model = AutoModelForCausalLM.from_pretrained(model_dir, use_cache=False)
+        self.tokenizer = AutoTokenizer.from_pretrained(model_dir, return_tensors="pt")
+        super().initialize(ctx)
+
+    def preprocess(self, requests):
```

Comment:
There's some code duplication between this PR and @HamidShojanazeri PR so now if you need to fix some bug you'd need to fix in as many large model backends as you've integrated, would prefer we refactor this into utils

Reply:
the two examples apply different model parallel library on the same model. @HamidShojanazeri pr demo pippy, here demo ds. that's why there are common parts to create model and tokenizer.